src/Application.java

```diff
@@ -0,0 +1,58 @@
+import sun.java2d.loops.TransformBlit;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+public class Application {
+
+    public void start() {
+        Transaction[] transactions = getData();
+        Transaction[] orderedTransactions = orderTransactions(transactions);
+        for (Transaction transaction : orderedTransactions) {
+            print(transaction);
+        }
+    }
+
+    public Transaction[] getData() {
```

Comment:
No reason for this method to be public - or even to be a method at all. Can just inline this one.